src/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,28 @@
+import s from './Cards.module.css';
+import React from 'react';
+import Card, { CardProps } from './Card/Card';
+
+interface CardsProps {
+  cards: CardProps[];
+}
+
+const CardList = (props: CardsProps) => {
+  const { cards } = props;
+  return (
+    <div className={s.cards}>
+      {cards.map((card) => (
+        <Card
+          id={card.id}
+          key={card.id}
+          tripName={card.tripName}
+          tripDate={card.tripDate}
+          tripType={card.tripType}
+          overnightStay={card.overnightStay}
+          tripImg={card.tripImg}
+        />
+      ))}
+    </div>
+  );
+};
+
+export default CardList;
```

Comment:
export default React.memo(CardList) if component have props it's preferable to use React.memo